packages/front-end/services/UserContext.tsx

```diff
@@ -189,10 +184,16 @@ export function UserContextProvider({ children }: { children: ReactNode }) {
     const members = currentOrg?.members;
     if (!members) return userMap;
     members.forEach((member) => {
-      userMap.set(member.id, member);
+      userMap.set(member.id, {
+        ...member,
+        userPermissions:
+          currentOrg?.currentUserPermissions && currentUser?.id === member.id
+            ? currentOrg?.currentUserPermissions
+            : undefined,
+      });
```

Comment:
This isn't needed anymore.  Below, instead of doing `user?.userPermissions`, you can just do `currentOrg?.currentUserPermissions`